commands/utils/params.go

```diff
@@ -321,6 +325,14 @@ func extractGitParamsFromEnv() (*Git, error) {
 	return &gitParams, err
 }
 
+// Endpoint should empty for default values or start with https scheme.
+func verifyValidApiEndpoint(endpoint *string) error {
+	if *endpoint != "" && !strings.HasPrefix(*endpoint, "https://") {
+		return errors.New("missing api endpoint scheme")
```

Comment:
You need to utilize `ErrMissingEnv` here. Returning a regular error will fail the process. Note that we don't want to fail on an empty `ApiEndpoint` because, in such cases, froggit-go automatically inserts the default API endpoint for each git provider.